### PR TITLE
Docs: Add Promoting Scenario

### PR DESCRIPTION
**Pull request by SteveLasker**
Adding the promotion scenario that includes trojan hose protection prior to import, and internal signature validation after the import

Note: With the shift to putting docs under the notaryproject.dev repo, this likely needs to move.
Staging to capture the docs for `notation` and `oras cp` iterations

### CODE REVIEW

File: docs/promoting-artifacts.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  export PRIVATE_IMAGE=${PRIVATE_REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local registry representing the Wabbit Networks **public** registry
+  ```bash
+  docker run -d -p ${PUBLIC_PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Run a local registry representing the ACME Rockets **private** registry
+  ```bash
+  docker run -d -p ${PRIVATE_PORT}:5000 ghcr.io/oras-project/registry:latest
```
Comment (iamsamirzon) [inline docs/promoting-artifacts.md:50]: Should this be ${PRIVATE_PORT}:5050

File: docs/promoting-artifacts.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- Create a test cert for the ACME Rockets Library key
+  ```bash
+  notation cert generate-test "acme-rockets.io-library"
+  ```
+- To support tag update scenarios, the image must be signed with a new signature and pushed with all artifact references
+- Sign the imported image, locally
+  ```bash
+  docker pull $PUBLIC_IMAGE
+  docker tag $PUBLIC_IMAGE $PRIVATE_IMAGE
+  docker notation sign $PRIVATE_IMAGE
```
Comment (iamsamirzon) [inline docs/promoting-artifacts.md:95]: does this notation sign command have to be preceded by "docker".  Is the assumption here that a notation plugin for Docker is present?

**Comment by SteveLasker** [comment]
As we're moving doc to the notaryproject.dev repo and site, just flagging this one to be redone and moved over.

**Comment by SteveLasker** [comment]
Just noting this will move to the https://github.com/notaryproject/notaryproject.dev repo

**Pull request closed without merging**